terminology.md

```diff
@@ -409,6 +409,16 @@ SDO is short for a Standards Developing Organization.
 
 [Wikipedia](https://en.wikipedia.org/wiki/Standards_organization)
 
+## Syntax budget
+
+### Definition
+
+The idea that there's an upper bound on how much syntactic complexity a language can reasonably contain, expressed as a metaphorical budget.
+
+A proposal without syntactic implications, such as the addition of a new method to a built-in, may be relatively uncontroversial, but a proposal that introduces new syntax is expected to prove that the cost is justified. This means both that (1) it addresses a significant developer need in a way that would not be reasonably achieved without new syntax and (2) the specific syntax being proposed does not obstruct another proposal expected to advance in the foreseeable future.
```

Comment:
```suggestion
A proposal without syntactic implications, such as the addition of a new method to a built-in, may be relatively uncontroversial, but a proposal that introduces new syntax is expected to meet a high bar to prove that the cost is justified. This means both that (1) it addresses a significant developer need in a way that would not be reasonably achieved without new syntax and (2) the specific syntax being proposed does not obstruct another proposal expected to advance in the foreseeable future.
```

Reply:
This seems okay, but I'm not sure I follow how it's changing the implication?

Reply:
Ah, I meant  "cost" relative to the syntax "budget" but that's not totally clear.

Reply:
Possibly "that the cost for the syntax budget", then?

Reply:
Updated with a similar phrase.